app/_src/quickstart/universal-demo.md

```diff
@@ -125,148 +131,140 @@ kuma-dp run \
         servicePort: 5000
         serviceAddress: 127.0.0.1
         tags: 
-          kuma.io/service: app
+          kuma.io/service: demo-app
           kuma.io/protocol: http
     admin:
-      port: 9902"
+      port: 9906"
 ```
 
-## Deploy the demo application
+### Run kuma-counter-demo app
 
-1. Run `redis` as a daemon on port 26379 and set a default zone name:
-  ```sh
-  redis-server --port 26379 --daemonize yes
-  redis-cli -p 26379 set zone local
-  ```
+1. With data planes running we can start our apps, first we will start and configure `Redis`:
```

Comment:
```suggestion
1. With the dataplane proxies running, we can start our apps, first we will start and configure `Redis`:
```

Reply:
We are distingwishing the dataplane proxy with the actual application here, so it better to add "proxy".